### PR TITLE
fix: skip questions on docker extension

### PR DESCRIPTION
**Pull request by jLopezbarb**
Signed-off-by: Javier López Barba <javier@okteto.com>

## Proposed changes
- If no manifest is found return err
- If no dev section can be inferred it should guide the user to create one

### CODE REVIEW

**Review by ifbyol — APPROVED** [review]

**Comment by jLopezbarb** [comment]
If no dev section is defined we show the logs of the full compose
Captura de pantalla 2022-05-09 a las 12 10 52
If dev section is defined we only show logs of the development services
Captura de pantalla 2022-05-09 a las 12 12 27

**Comment by pchico83** [comment]
> If no dev section is defined we show the logs of the full compose Captura de pantalla 2022-05-09 a las 12 10 52 If dev section is defined we only show logs of the development services Captura de pantalla 2022-05-09 a las 12 12 27

Let's always show logs for all services, it gives a more consistent behavior

File: cmd/up/up.go [hunk 1]
```diff
@@ -142,7 +142,7 @@ func Up() *cobra.Command {
 			}
 			os.Setenv(model.OktetoNameEnvVar, oktetoManifest.Name)
 
-			if len(oktetoManifest.Dev) == 0 {
+			if len(oktetoManifest.Dev) == 0 && !upOptions.DockerDesktop {
```
Comment (rberrelleza) [inline cmd/up/up.go:145]: @jLopezbarb can we make this more general? We need the same behavior for the vscode extension

Comment (pchico83) [reply cmd/up/up.go:145]: @rberrelleza this one in particular only applies to the docker extension, and it's only done on this feature branch.
what would you need for the vscode extension?

Comment (jmacelroy) [reply cmd/up/up.go:145]: What happens when this feature is merged to main?

Comment (pchico83) [reply cmd/up/up.go:145]: @jmacelroy we will need to evaluate how to do it. There are two options:
- Move this logic to the Docker Extension itself
- Make this logic more generic and implement the detach flag

Comment (rberrelleza) [reply cmd/up/up.go:145]: I think that making a generic option for "don't ask any questions and just assume defaults" is very useful (we need that for the okteto vscode extension for the same reason than docker desktop).  But I think that's a different requirement from a "detached mode". To me, the "ask no questions" mode sounds like a requirement for detached, but they not need to be tightly coupled. You might still want the process at the forefront, just without requiring any input.

Comment (pchico83) [reply cmd/up/up.go:145]: Makes sense. I would do that on a separate PR and analyze where we are prompting for questions in the current codebase.

**Pull request merged**